mainnet.yaml

```diff
@@ -29,9 +29,12 @@ files:
 - deployments/cardano-explorer-env-production.nix
 - deployments/report-server.nix
 - deployments/report-server-env-production.nix
+- deployments/monitoring.nix
```

Comment:
now that `require` is in use, i dont think `monitoring.nix` has to be listed here anymore, it gets pulled in by `monitoring-env-staging.nix`
also, this is mainnet, why is it loading `monitoring-env-staging.nix`?